Competition2017/src/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -0,0 +1,52 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToPositionCommand extends BaseCommand {
+
+    private final DriveSubsystem driveSubsystem;
+    private final PIDManager travelManager;
+    
+    private final DoubleProperty distanceToleranceInches;
+    private double targetPosition;
+    
+    @Inject
+    public DriveToPositionCommand(DriveSubsystem driveSubsystem, XPropertyManager propManager) {
+        this.driveSubsystem = driveSubsystem;
+        this.requires(driveSubsystem);
+        this.travelManager = new PIDManager("Drive to position", propManager, 0.1, 0.1, 0.1);
```

Comment:
Unless you already know that 0.1 is a good constant for the I term, it's recommended to have it be at 0 (since it can lead to extreme robot behavior).